### PR TITLE
Amazon services: remove dynamodb mentions from common config beans

### PR DESCRIPTION
**Pull request by marcinczeczko**
Removed dynamodb mentions from the common config beans.

### CODE REVIEW

**Comment by geoand** [comment]
Formatting is needed:

```
2020-05-14T08:37:16.6300949Z Caused by: org.apache.maven.plugin.MojoFailureException: File '/home/runner/work/quarkus/quarkus/extensions/amazon-services/common/runtime/src/main/java/io/quarkus/amazon/common/runtime/SyncHttpClientConfig.java' has not been previously formatted.  Please format file and commit before running validation!
```

**Comment by gsmet** [comment]
I just force pushed a formatting fix.

File: extensions/amazon-services/common/runtime/src/main/java/io/quarkus/amazon/common/runtime/AwsConfig.java [hunk 1]
```diff
@@ -11,10 +11,10 @@
 
     // @formatter:off
     /**
-     * An Amazon Web Services region that hosts DynamoDB.
+     * An Amazon Web Services region that hosts given service.
```
Comment (geoand) [inline extensions/amazon-services/common/runtime/src/main/java/io/quarkus/amazon/common/runtime/AwsConfig.java:14]:
```suggestion
     * An Amazon Web Services region that hosts the given service.
```

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
Thanks!

**Comment by gsmet** [comment]
Squashed the commits. Let's wait for CI and merged.

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**